api/src/main/java/org/apache/any23/extractor/Extractor.java

```diff
@@ -39,22 +38,6 @@
 
     /**
      * This interface specializes an {@link Extractor} able to handle
-     * {@link java.io.InputStream} as input format.
-     */
-    public interface ContentExtractor extends Extractor<InputStream> {
```

Comment:
@jgrzebyta yes this is correct... we do not always wish to assume that the input is structured in XML or a subset thereof... syntax-strict extractors are prone to breakage. Our aim in Any23 should be to provide flexibility in the extraction logic rather than a strict, fragile extraction logic.